docs/piece-pages.md

```diff
@@ -1,25 +1,20 @@
 ---
-title: "Piece pages"
+title: "Piece Pages"
 ---
 
-# Piece pages in A3
+# Piece Pages
 
-:::  tip Note:
-**"What's a piece page?"**
-<!-- I realize that for alpha 1 our audience is more familiar with Apostrophe, but this is a topic that comes up even among people who know a lot about Apostrophe, so I'd like this note to stay in. Let's work on making it as clear as possible. Tom -->
-
-The most intuitive example is a blog: if the individual [piece](pieces.md) is one blog post, then the piece page is the blog's home page, where all of the posts can be discovered, paginated, filtered and explored. In addition,  "piece pages" are responsible for serving the individual webpage for each piece.
+The most intuitive example of a piece page is an index of products: if the individual [piece](pieces.md) is one product, then the piece page is the product home page, where all of the posts can be discovered, paginated, filtered and explored. In addition, piece pages are responsible for serving the individual page for each piece.
 
 If the project has a piece module called `product`, then that module provides a way to create, edit, manage and query pieces. If the project has a piece page module called `product-page`, then that module provides a way to browse and view the pieces.
 
-In general piece modules are concerned with *editing and APIs*, while piece page modules are concerned with *browsing as part of a website.* For developers familiar with the "model / view / controller" pattern: **piece modules are the model layer for your content type, while piece page modules are the view layer.**
+In general piece modules are concerned with *editing and APIs*, while piece page modules are concerned with *browsing as part of a website or application.* For developers familiar with the "model / view / controller" pattern: **piece modules are the model layer for your content type, while piece page modules are the view layer.**
```

Comment:
It really is specifically a website in this case. Applications will probably use the REST APIs.